Tests/Recurly/Transaction_Test.php

```diff
@@ -17,6 +17,7 @@ public function testGetTransaction() {
     $this->assertInstanceOf('Recurly_Stub', $transaction->account);
     $this->assertInstanceOf('Recurly_Stub', $transaction->invoice);
     $this->assertInstanceOf('Recurly_Stub', $transaction->subscription);
+    $this->assertInstanceOf('Recurly_FraudInfo', $transaction->fraud);
```

Comment:
Since we don't have any fraud info specific tests can expand this a little more to check the decision and score?